### DISCUSSION TITLE
TypeError: function is not a constructor (evaluating 'super(...arguments)') on iOS 13

### DISCUSSION THREAD

**Discussion by brianyuen** [google/model-viewer#3373] [Q&A] [answered]
### Description

Hi, currently testing out model viewer on older devices and came across a TypeError while visiting modelviewer.dev. None of the models are showing up in mobile Safari on iOS 13.6 and getting this in the console `TypeError: function is not a constructor (evaluating 'super(...arguments)')` I'm currently using iOS Simulator on XCode 13.3 to produce the error.

[Screen grab here](https://imgur.com/a/LQVgBk6)

#### Live Demo

https://modelviewer.dev

### Version

- Whatever is being used as of today on modelviewer.dev

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ X ] Safari on iOS 13.6

### OS

- [ ] Android
- [ X ] iOS 13.6
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment, marked as answer]
We only officially support (i.e. run our automated tests on) the last two versions of iOS, which are currently 14 and 15. They have enough WebGL bugs to work around as is, we just can't support all of their legacy.

**Comment by samuelndm** [comment]
@brianyuen Have you been able to found a solution for this error?

**Reply by elalish** [reply]
Are you also testing on iOS 13? That version is not supported.

**Reply by samuelndm** [reply]
@elalish thanks
